frontend/src/pages/modelServing/screens/projects/ModelMeshSection/ServingRuntimeDetails.tsx

```diff
@@ -25,9 +25,11 @@ const ServingRuntimeDetails: React.FC<ServingRuntimeDetailsProps> = ({ obj, isvc
   const enabledAcceleratorProfiles = acceleratorProfile.acceleratorProfiles.filter(
     (ac) => ac.spec.enabled,
   );
-  const container = obj.spec.containers[0]; // can we assume the first container?
+  const resources = obj.spec.containers[0].resources || isvc?.spec.predictor.model.resources;
```

Comment:
We need to update our types -- `model` is not required, and `resources` are not required

Reply:
Yes, I'm gonna update the type here and include the fix in this PR given that it's being carried over

Reply:
Ok, the PR is growing more than it should, I’m gonna open a separate PR and bring it to this sprint.

Reply:
https://issues.redhat.com/browse/RHOAIENG-6794